### ISSUE TITLE
Add Description to News/Latest items

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/network#792]
Add a few lines for either a summary or editorial commentary. The goal: provide more context on stories and guidance to visitors.

The CMS currently has a Excerpt field that we can leverage. Check staging to see if it's hooked up to the frontend. We might have also had this in design spec. Confirm the quality, but no need to start from scratch.

**Comment by taisdesouzalessa** [comment]
Here are the screenshots with these excerpts.

![3vs2lines](https://user-images.githubusercontent.com/22157921/30939626-e07de63c-a392-11e7-94f8-43167f473d99.jpg)

2-line summaries (image 2) work better than 3-line (image 1). They are easier to scan and convey the message. The character count is 145, which is good to tell a little short story. 

For instance (example taken from The New York Times): 
"A pair of figure skaters represent the best chance for the country to have athletes at the 2018 Winter Games in South Korea. " - 125 characters. 

Maybe it would be good to get Sydette's opinion here in terms of character counting?

My only concern is for screens smaller than 976px. This layout have a big gap on that screen size:
![screensize-small-equal976](https://user-images.githubusercontent.com/22157921/30940571-53846fd6-a396-11e7-8a5d-ef0561400a8d.jpg)

**Comment by xmatthewx** [comment]
Sounds. Let's keep in mind that we have limited time to implement this pre-mozfest, so let's look for a simple solution that gets the job done. We might have to make some tradeoffs.

**Comment by taisdesouzalessa** [comment]
Mockups posted here: https://github.com/mozilla/network/issues/791#issuecomment-333688967

**Comment by xmatthewx** [comment]
@taisdesouzalessa – did you test whether this is already implemented on staging?

Let's put our specs here for implementation: max word count, font-size, spacing above.

**Comment by taisdesouzalessa** [comment]
Hey @xmatthewx I talked to @alanmoo about it and it seems that although we have the "excerpt" field on the CMS it is not displaying once the page is published. 

I'll prepare the specs and post them here.

**Comment by taisdesouzalessa** [comment]
One question that just came to mind, @xmatthewx: will this change in the homepage affect the News page as well? All the news on [this page](https://network.mozilla.org/news/) will have excerpts too? Not a big deal if they have, it will just make the page even longer. Maybe we should consider adding a filter and pagination in future releases?

**Comment by alanmoo** [comment]
From a technical standpoint, the News page doesn't have to show the excerpts.

**Comment by xmatthewx** [comment]
Cool, cool. I'll file a ticket to figure out excerpts on News page later. Let's keep this scoped to the homepage.

**Comment by taisdesouzalessa** [comment]
Here are the specs:
1. JPEG with the final look of this section (I did this mockup on the browser so it is following the already implemented rules):
<img width="832" alt="image" src="https://user-images.githubusercontent.com/22157921/31199842-8c1c8be0-a90d-11e7-950a-b089602734bf.png">

2. Character count (max): 146

3. For spacing between date and text excerpt, I just added a `<p></p>` after the date, so the spacing is what we already have there (16px - yellow part of the image below)
<img width="373" alt="image" src="https://user-images.githubusercontent.com/22157921/31200045-3e78ca56-a90e-11e7-96e5-eb3a1dd648e8.png">
